src/main/java/nl/knaw/dans/easy/sword2examples/Common.java

```diff
@@ -118,11 +127,11 @@ public static CloseableHttpClient createHttpClient(URI iri, String uid, String p
         return HttpClients.custom().setDefaultCredentialsProvider(credsProv).build();
     }
 
-    public static CloseableHttpResponse sendChunk(DigestInputStream dis, int size, String method, URI iri, String filename, String mimeType, CloseableHttpClient http, boolean inProgress) throws Exception {
+    public static CloseableHttpResponse sendChunk(DigestInputStream dis, int size, String method, IRI iri, String filename, String mimeType, CloseableHttpClient http, boolean inProgress) throws Exception {
```

Comment:
This `IRI` is also converted to a `URI`, as well as all its uses in the other classes. Why use this `IRI` anyway?